### ISSUE TITLE
partman with native partition create child table pk and index only in default tablespace

### ISSUE THREAD

**Issue by dnlcrt** [pgpartman/pg_partman#223]
Hello,
I'm using PG10 with native partition and pg_partman 3.1.3.
I have template table with index and pk created on tablespace only for index, but 
when child tables are created all index use default tablespace.

Is it possible to modify pg_partman to create child table index with the same tablespace used in the template table?

Thanks

**Comment by keithf4** [comment]
Sure, I can look into that. Thanks for reporting the issue!

**Comment by keithf4** [comment]
This should fixed in next release. Tablespaces can actually be defined on the index level itself. So instead of basing the tablespace on what the template table is set to, I have it checking the index itself. So you can actually have the index tablespace be different than the table if needed.

**Comment by iwarford** [comment]
There appears to be a bug in this fix.  When running against a template table with indexes on PG10, I get:

```
ERROR:  unterminated format() type specifier
CONTEXT: PL/pgSQL function inherit_template_properties(text,text,text) line 108 at assignment
SQL statement "SELECT partman.inherit_template_properties(p_parent_table, v_parent_schema, v_partition_name)"
PL/pgSQL function create_partition_time(text,timestamp with time zone[],boolean,boolean) line 213 at PERFORM
PL/pgSQL function partman.create_parent(text,text,text,text,text[],integer,text,text,boolean,text,text,boolean,text,boolean,boolean) line 413 at assignment
DETAIL:
HINT: For a single "%" use "%%".
CONTEXT: PL/pgSQL function create_partition_time(text,timestamp with time zone[],boolean,boolean) line 433 at RAISE
PL/pgSQL function partman.create_parent(text,text,text,text,text[],integer,text,text,boolean,text,text,boolean,text,boolean,boolean) line 413 at assignment
DETAIL:
HINT:
CONTEXT:  PL/pgSQL function partman.create_parent(text,text,text,text,text[],integer,text,text,boolean,text,text,boolean,text,boolean,boolean) line 670 at RAISE
```

It looks like there are two format() statements that should be %I and are just %.  Attached diff.

[inherit_template_properties.sql.diff.txt](https://github.com/pgpartman/pg_partman/files/2172021/inherit_template_properties.sql.diff.txt)

**Comment by keithf4** [comment]
Thanks. Will try and get that fixed ASAP.

**Comment by keithf4** [comment]
@iwarford Should be fixed in v3.2.1. Thanks again for reporting the issue!
